app/agent/mysql.go

```diff
@@ -89,6 +89,7 @@ func (h *MySQLHandler) GetAll() ([]proto.Agent, error) {
 
 	agents := []proto.Agent{}
 	var version sql.NullString
+	var deleted mysqlDriver.NullTime
```

Comment:
There was an issue with NULL time in db, that was leading this code to panic in tests because you can't scan NULL to time.Time.
Probably with real life data this
a) doesn't happen because deleted always has value different than NULL
b) doesn't happen because this GetAll() method is never used by API
c) it happens but we don't know